applications/OptimizationApplication/python_scripts/utilities/opt_convergence.py

```diff
@@ -0,0 +1,32 @@
+import KratosMultiphysics as Kratos
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+
+def CreateConvergenceCriteria(parameters: Kratos.Parameters, optimization_problem: OptimizationProblem):
+    type = parameters["type"].GetString()
+    if type == "max_iter":
```

Comment:
could you please add L2 norm for the sake of completner and simplicity. Only for shape optimization it is diffciult to get L2 norm close to zero, For topology and thickness optimization, we easily get converged results with L2 norm criterion.

Reply:
I have added L2 norm convergence criteria